changelogs/fragments/685-zabbix-script.yml

```
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - zabbix_script module added
+  - (https://github.com/ansible-collections/community.zabbix/issues/634)
```

```suggestion
  - zabbix_script module added (https://github.com/ansible-collections/community.zabbix/issues/634)
```

has to be single line, otherwise changelog format will get wonky and require manual fixes

---

does file name matter ? 685-zabbix-script.yml should be 689-zabbix-script.yml I think.

---

it doesnt